### PR TITLE
Random test case failure fix

### PR DESCRIPTION
**Pull request by pavan187**
Randomly device state change online is failing.

image

### CODE REVIEW

File: test/integration/scripts/execute.sh [hunk 1]
```diff
@@ -26,7 +26,8 @@ go build cmd/edge_core.go
 #dynamically append testManager Module before starting integration test.
 sed -i 's/dbTest/dbTest, testManager/g' conf/modules.yaml
 #restart edge_core after appending testManager Module.
-pkill edge_core
+sudo pkill edge_core
+sleep 2s
```
Comment (m1093782566) [inline test/integration/scripts/execute.sh]: Please comment why we sleep 2s and is 2s enough?

File: test/integration/modules/edgecore/eventbus/eventbus_test.go [hunk 2]
```diff
@@ -19,12 +19,10 @@ package integration
 import (
 	"database/sql"
 	"encoding/json"
-	"os"
-	"path/filepath"
-	"time"
-
 	"github.com/kubeedge/kubeedge/pkg/devicetwin/dtcommon"
 	"github.com/kubeedge/kubeedge/test/integration/utils/common"
+	"os"
```
Comment (m1093782566) [inline test/integration/modules/edgecore/eventbus/eventbus_test.go]: Put these imported package in suitable place?

File: test/integration/scripts/execute.sh (hunk 1 above)
Comment (pavan187) [reply test/integration/scripts/execute.sh]: @m1093782566 , as suggested gave a comment on top.

File: test/integration/modules/edgecore/eventbus/eventbus_test.go (hunk 2 above)
Comment (pavan187) [reply test/integration/modules/edgecore/eventbus/eventbus_test.go]: @m1093782566 , OK..

**Comment by m1093782566** [comment]
ping @sids-b for review.

**Pull request merged**